notebooks/run_model_in_notebook.ipynb

```diff
@@ -15,11 +15,9 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    import PyStemmusScope.iostreamer
</pre><p>we can replace it by <code>import PyStemmusScope</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/EcoExtreML/STEMMUS_SCOPE_Processing/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Revised.</p>